### PR TITLE
Feat: add isCreatable option to SelectField

### PR DESCRIPTION
**Pull request by danad02**
Needed for some cases in MC.

### CODE REVIEW

File: packages/ui/src/SelectField.tsx [hunk 1]
```diff
@@ -201,30 +204,58 @@ export const SelectField = <V,>({
...
-          components={{
-            DropdownIndicator,
-            ClearIndicator,
-            Input,
-            MultiValue,
-          }}
-          {...rest}
-        />
+        {isCreatable ? (
+          <ReactSelectCreatable<SelectFieldOption<V>>
```
Comment (aigoncharov) [inline packages/ui/src/SelectField.tsx:208]: nit: They have the same props, right? Should we then do something like this?
```tsx
const SelectComponent = isCreatable ? ReactSelectCreatable : ReactSelect
...
return <SelectComponent  .../>
```

**Review by aigoncharov — APPROVED** [review]

File: packages/ui/src/SelectField.tsx (hunk 1 above)
Comment (danad02) [reply packages/ui/src/SelectField.tsx:208]: Creatable has the same props as classic select and **some additional** props. I ended up not using any of the extra props, so for now we can change the implementation to look as you suggest.

Comment (aigoncharov) [reply packages/ui/src/SelectField.tsx:208]: @danad02 alternatively we could create a single object with all props and apply it to both components. It is probably a better approach.

Comment (aigoncharov) [reply packages/ui/src/SelectField.tsx:208]:
```tsx
const props = {}
...
return isCreatable ? <CreatableSelect {...props} /> : <Select {...props} />
```

**Pull request merged**